CHANGES_NEXT_RELEASE

```diff
@@ -23,6 +23,7 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Fix: proper response for 'GET /v2/entities/<id>?attrs=<attrName>' and 'GET /v2/entities/<id>/attrs?attrs=<attrName>' when <attrName> doesn't exist (#2241)
```

Comment:
Note that I have pseudo-randomly put the new entry in the middle of CNR file, not the end. If each developer do the same, this could minimize the probability of the typical merge conflict due to two PRs editing at the end of the file. At the end I always reorder the line before releasing.

NTC (just a "trick" I'd like to share :)